device-types/Ubiquiti/ER-X.yaml

```diff
@@ -4,7 +4,7 @@ model: ER-X
 slug: er-x
 u_height: 1
 is_full_depth: false
-comments: '[EdgeRouter X - Advanced Gigabit Ethernet Router](https://www.ui.com/edgemax/edgeswitch-10x/)'
+comments: '[EdgeRouter X - Advanced Gigabit Ethernet Router](https://store.ui.com/collections/operator-edgemax-routers/products/edgerouter-x)'
```

Comment:
I actually think this should be the datasheet: https://dl.ui.com/datasheets/edgemax/EdgeRouter_X_DS.pdf

Reply:
Adjusted